webcompat/issues.py

```diff
@@ -23,12 +23,17 @@
 PRIVATE_REPO_URI = app.config['PRIVATE_REPO_URI']
 
 
-def unmoderated_issue():
+def moderation_template():
     """Gets the placeholder data to send for unmoderated issues."""
-    # TODO: Replace this with something meaningful.
-    # See https://github.com/webcompat/webcompat.com/issues/3137
-    summary = 'Placeholder in-moderation title.'
-    body = 'Placeholder in-moderation body.'
+
+    summary = """In the moderation queue."""
```

Comment:
nit:
* The `"""` are not required here as it is only one line.  
* Also we use (for now) single quote styles aka `'bla'` or `'''bla'''` except for docstring. We might change one day, but for now it's what we did.

Reply:
thanks! fixed this